### PR TITLE
Update kubernetes 1.14.5 (CVE-2019-11247)

### PR DESCRIPTION
**Pull request by cornelius-keller**

### CODE REVIEW

File: service/controller/v23patch1/version_bundle.go [hunk 1]
```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.14.5 (CVE-2019-11247)",
```
Comment (MarcelMue) [inline service/controller/v23patch1/version_bundle.go]: We should put a link into this description, see here for example: https://github.com/giantswarm/kvm-operator/blob/master/service/controller/v23/version_bundle.go#L12

**Comment by schjan** [comment]
I don't know how to comment unchanged files but you miss one place to update the versionbundle at https://github.com/giantswarm/kvm-operator/blob/kubernetes-1.14.5/service/controller/v23patch1/cloudconfig/cloudconfigtest/cloudconfigtest.go

File: service/controller/v23patch1/version_bundle.go (hunk 1 above)
Comment (cornelius-keller) [reply service/controller/v23patch1/version_bundle.go]: fixed

**Comment by cornelius-keller** [comment]
> I don't know how to comment unchanged files but you miss one place to update the versionbundle at https://github.com/giantswarm/kvm-operator/blob/kubernetes-1.14.5/service/controller/v23patch1/cloudconfig/cloudconfigtest/cloudconfigtest.go
It seems somebody forgot this already the last time. Did not see it because it was still on `v_4_4_0`

**Pull request merged**